### PR TITLE
html-forms - Vorleak Yek 

### PR DESCRIPTION
**Pull request by vorleakyek**
https://user-images.githubusercontent.com/76607726/231670834-24a574df-a0f0-42b3-b10b-da670f89dd1a.mp4

### CODE REVIEW

File: html-forms/index.html [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+      <p>Favorite Language (choose one)</p>
+
+      <input type="radio" id="html" name="language">
+      <label for="html">HTML</label>
+
+      <input type="radio" id="css" name="language">
+      <label for="css">CSS</label>
+
+      <input type="radio" id="javascript" name="language">
+      <label for="javascript">JavaScript</label>
```
Comment (kurtyazdizadeh) [inline html-forms/index.html]: let's give these `input`s a `value` attribute to so we will know what selection they made (value is what data will be sent on submit events)

**Review by kurtyazdizadeh — CHANGES_REQUESTED** [review]
One small change, see comments

File: html-forms/index.html (hunk 1 above)
Comment (vorleakyek) [reply html-forms/index.html]: Okay, I added the value attribute to those input elements. Thanks!

**Review by kurtyazdizadeh — APPROVED** [review]
Approved 👍

**Pull request merged**